python/tvm/meta_schedule/testing/torchbench/run.py

```diff
@@ -0,0 +1,301 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+This script is for benchmarking TVM performance on models from TorchBench.
+It uses the torchdynamo as the frontend to ingest models into TVM, and it also
+leverages the benchmark util from torchdynamo.
+
+To setup environment:
+1. Clone https://github.com/pytorch/torchdynamo and https://github.com/pytorch/benchmark 
+   to the parent directory of TVM.
+2. Follow the install instruction of torchdynamo and pytorch/benchmark.
+"""
+
+import argparse
+import functools
+import logging
+import time
+
+import numpy as np
+import torch
+from scipy.stats import ttest_ind
+
+import tvm
+from tvm import meta_schedule as ms
+from tvm.contrib import graph_executor
+from tvm.meta_schedule.testing.torchbench.util import load_torchdynamo_benchmark_runner, same, timed
+from tvm.support import describe
+
+runner = load_torchdynamo_benchmark_runner()
+import torchdynamo
+
+
+def parse_args():
+    args = argparse.ArgumentParser()
+
+    args.add_argument(
+        "--disable-tuning",
+        action="store_true",
+        help="""
+        Run the script without tuning. It compiles the model using
+        existing database at workdir.
+        """,
+    )
+    args.add_argument(
+        "--disable-benchmark",
+        action="store_true",
+        help="""
+        Skip running the benchmark and exit right after tuning finishes.
+        """,
+    )
```

Comment:
merge into `--mode`. let's also consider the case where database is empty without any trial